vscode/src/documentation.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { getCompilerWorker } from "qsharp-lang";
+import { isQsharpDocument } from "./common";
+import { getTarget } from "./config";
+import { Uri, window } from "vscode";
+import { loadProject } from "./projectSystem";
+import { sendMessageToPanel } from "./webviewPanel";
+
+export async function showDocumentationCommand(extensionUri: Uri) {
+  // Reveal panel an show 'Loading...' for immediate feedback.
+  sendMessageToPanel(
+    "documentation", // This is needed to route the message to the proper panel
```

Comment:
I changed the string here to be consistent across the panel and the calling code, else reloading the VS Code window doesn't deserialize and reload the state (i.e. old content) correctly.

Reply:
Updated PR description and removed this as not yet implemented.